### DISCUSSION TITLE
Docker Desktop alternatives compatible with Warden on Mac

### DISCUSSION THREAD

**Discussion by ihor-sviziev** [wardenenv/warden#807] [Help & Support]
Hi,

Historically, I've been using Docker Desktop on a Mac computer with Apple Silicon CPU and paid for the Pro version, and Warden works perfectly with it.
Now, they sent an email stating that pricing will be reviewed soon (https://www.docker.com/blog/november-2024-updated-plans-announcement/ https://www.docker.com/pricing/november-2024-subscription-changes/), which led me to wonder if I should stay on it or switch to an alternative.

I also know some people use Warden with OrbStack and have issues like https://github.com/wardenenv/warden/issues/718.

To clarify:
- I don't need a UI for Docker. I'm entirely OK with having a command-line-only tool like it works on Linux. A
- I don't need a docker registry, which will be included in the pricing
- I don't need to have built-in security scans or any other service for docker

I previously tried Portainer and a few other options, but Warden's local DNS server didn't work there.

Does anyone use any other versions of docker installations for Mac?

**Comment by dudzio12** [comment]
After migration from Docker Desktop to Orbstack I observed huge boost in performance. I'm looking forward to close https://github.com/wardenenv/warden/pull/749, since Orbstack is faster and more stable by itself in file sync then mutagen. I worked with dirty-patched warden, without mutagen for months. Only after latest warden update I bring back mutagen, but I will patch it one more time, because mutagen sync is a nightmare for me.

**Comment by navarr** [comment]
I've been using Warden with Orbstack and Mutagen without issue on an M3 macbook pro.

**Reply by dudzio12** [reply]
I have M2 Pro and 2-3 large Magento projects running on daily basics. I'll try to finish the PR I mentioned because it looks abandoned, so you could test this approach yourself :)
